access/types/forms.py

```diff
@@ -353,6 +353,10 @@ def grade(self):
                     if gname not in error_groups:
                         error_groups.append(gname)
             g += 1
+
+        if self.exercise["feedback"]:
+            points = self.exercise.get("max_points")
```

Comment:
The default in `get` is None if it is not specified, so you should do `self.exercise.get("max_points", 0)`.